plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -113,6 +113,10 @@ public class HiveConfig
 
     private String orcLegacyTimeZone = TimeZone.getDefault().getID();
 
+    // This is a feature flag for Ion native trino integration.
+    // Default value is false and requires explicitly setting it to true to enable Ion native trino integration.
```

Comment:
I assume that at some point in the future it will flip to default true. Does that sound right? Is it worth documenting here?

Reply:
Yes, I think thats a good idea. I will add some comment for it.